package.json

```diff
@@ -24,17 +24,15 @@
         "/src"
     ],
     "dependencies": {
-        "@react-native-picker/picker": "^1.8.3",
+        "@react-native-picker/picker": ">2.4.0",
```

Comment:
Can this be `^2.4.0` instead? The expo-approved version for API 45 is 2.4.0 but this syntax accepts only 2.4.1 and above.